packages/driver/src/cy/actionability.ts

```diff
@@ -53,9 +54,127 @@ const getPositionFromArguments = function (positionOrX, y, options) {
   return { options, position, x, y }
 }
 
+const ensureElDoesNotHaveCSS = ($el, cssProperty, cssValue, name: string, onFail) => {
+  const el = $el[0]
+  const win = $dom.getWindowByElement(el)
+  const value = win.getComputedStyle(el)[cssProperty]
+
+  if (value === cssValue) {
+    const elInherited = $elements.findParent(el, (el, prevEl) => {
+      if (win.getComputedStyle(el)[cssProperty] !== cssValue) {
+        return prevEl
+      }
+    })
+
+    const element = $dom.stringify(el)
+    const elementInherited = (el !== elInherited) && $dom.stringify(elInherited)
+
+    const consoleProps = {
+      'But it has CSS': `${cssProperty}: ${cssValue}`,
+    }
+
+    if (elementInherited) {
+      _.extend(consoleProps, {
+        'Inherited From': elInherited,
+      })
+    }
+
+    $errUtils.throwErrByPath('dom.pointer_events_none', {
+      onFail,
+      args: {
+        cmd: name,
+        element,
+        elementInherited,
+      },
+      errProps: {
+        consoleProps,
+      },
+    })
+  }
+}
+
+const ensureElIsNotAnimating = ($el, coords = [], threshold, name: string) => {
+  const lastTwo = coords.slice(-2)
+
+  // bail if we dont yet have two points
+  if (lastTwo.length !== 2) {
+    $errUtils.throwErrByPath('dom.animation_check_failed')
```

Comment:
nit: future PR, not here -- it might be nice to update these error keys some day to reflect where they actually live.